README.md

```diff
@@ -1,2 +1,64 @@
-# MC-Parallel-Functions
-Parallel Functions Morning Challenge
+**Author**: [@des-des](https://github.com/des-des)  
+
+**Maintainer**: TBC
+## Parallel Functions Morning Challenge
+
+When making API calls, exactly how quickly you receive a response can depend on the speed of your connection, connection latency, and the performance of the API that you're calling.
+When making several API calls, or reading data from several files at the same time, the order in which each task is completed will depend on how quickly the response/data can be retrieved/extracted, which may not always be the same order that the tasks were called.
+
+Parallel functions can help to run several tasks at the same time, and return the values of those tasks in the order they were called.
+
+Below is an array of functions (the ```tasks``` array), each one set to run at different delays (kind of like the delays you might experience when retrieving data form an API).
+
+Complete the 'parallel' function. It should fire all the tasks at the same time, and invoke the final callback when they are all done.
+
+In case of an error, the final callback should be fired immediately.
+
+The callback accepts two arguments which are error and an array of task results.
+
+TIP: You might want to use [Repl.it](https://repl.it/languages/javascript) to complete this challenge so you can see whats happening in your code.
+
+While you're working on this, think about when this function might be useful (in the context of what you've been studying this week).
+
+More info:
```

Comment:
These are all in a single line ([check the rendered output](https://github.com/foundersandcoders/mc-parallel-functions/blob/e4bac31c00909b915ced843416eb132d0d955be1/README.md)). If you want them to be new-lines, you need to either add two spaces at the ends of the lines, or put in an additional new-line